examples/react-relay/src/__generated__/IndexPageContainerMessagesQuery.graphql.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * @generated SignedSource<<e3c801c4201d1daae77623fc34fbaae4>>
+ * @lightSyntaxTransform
+ * @nogrep
+ */
+
```

Comment:
these generated files are specific to react-relay?

Reply:
Yeah these are generated by Relay. I will provide a `script` and add a note to README about this. Good spot.